### ISSUE TITLE
vim Integration with rg is broken

### ISSUE THREAD

**Issue by supermarin** [lotabout/skim#51]
If a SKIM_DEFAULT_COMMAND is specified, skim breaks after fuzzy selecting a file with:
```
Too many arguments for function: 9
[Process exited 0]
```
The interesting part is that the number `9` increases by 2 each time you try to fuzzy open a file, but starts at 9.
I'll take a look later into this if I find time, but any pointers appreciated.

**Comment by renstrom** [comment]
Perhaps related to these?

- https://github.com/junegunn/fzf/issues/765
- https://github.com/junegunn/fzf/issues/769

**Comment by supermarin** [comment]
Yep. After seeing @justinmk's comment, this patch fixes things.. it seems like there's an unused parameter in `on_exit` needed, or the callsite is passing an extra param

``` diff
diff --git a/plugin/skim.vim b/plugin/skim.vim
index 517dbff..347176a 100644
--- a/plugin/skim.vim
+++ b/plugin/skim.vim
@@ -341,7 +341,7 @@ function! s:execute_term(dict, command, temps) abort
       endif
     endif
   endfunction
-  function! skim.on_exit(id, code)
+  function! skim.on_exit(id, code, wat)
     if s:getpos() == self.ppos " {'window': 'enew'}
       for [opt, val] in items(self.winopts)
         execute 'let' opt '=' val
```

**Comment by supermarin** [comment]
Fixed in #52
